checks.d/tokumx.py

```diff
@@ -424,7 +424,11 @@ def collect_metrics(self, instance, server, conn, db, tags):
                 db_tags = list(tags)
                 db_tags.append('db:%s' % dbname)
                 db = conn[dbname]
-                stats = db.command('dbstats')
+                try:
+                    stats = db.command('dbstats')
+                except:
```

Comment:
Hmm, or maybe I need to use a pattern like [this](https://github.com/DataDog/dd-agent/blob/master/checks.d/tokumx.py#L380)?